### ISSUE TITLE
Error when loading pythreejs

### ISSUE THREAD

**Issue by jhoydis** [vidartf/ipydatawidgets#57]
It seems that pythreejs (v4.2.1) does not load anymore due to a missing attribute in ipydatawidgets (4.3.3).

  ```
File ".../lib/python3.8/site-packages/ipydatawidgets/ndarray/union.py", line 59, in subclass_init
    cls._instance_inits.append(self.instance_init)
AttributeError: type object 'DataImage' has no attribute '_instance_inits'
```

When I force the version of  ipydatawidgets to 4.3.2 the error disappears.

**Comment by gattia** [comment]
Same issue is occurring for itkwidgets: 

https://github.com/InsightSoftwareConsortium/itkwidgets/issues/621
https://github.com/InsightSoftwareConsortium/itkwidgets/issues/615

Forcing to <4.3.3 fixes the error.

**Comment by vidartf** [comment]
Sorry for taking so long to get back on this. The "fix" in 4.3.3 was to add compatibility with latest traitlets. I should have made it backwards compatible. A fix is in #59 now, and I hope to cut a new patch release with it later today.
